README.md

```
@@ -41,7 +41,9 @@ directory listing.
 
 Basic usage:
 
-    ./httpdirfs -f --cache $URL $MOUNT_POINT
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)